app/controllers/ncr/work_orders_controller.rb

```diff
@@ -29,7 +29,7 @@ def update
 
       super
 
-      if @model_changing && !@model_instance.emergency # skip approvals if emergency
+      unless skip_state_update?
```

Comment:
Why can't the method name explain the why!?

On Monday, October 19, 2015, Peter Karman <notifications@github.com> wrote:

> In app/controllers/ncr/work_orders_controller.rb
> <https://github.com/18F/C2/pull/715#discussion_r42449803>:
>
> > @@ -29,7 +29,7 @@ def update
> >
> >        super
> >
> > -      if @model_changing && !@model_instance.emergency # skip approvals if emergency
> > +      unless skip_state_update?
>
> This feels like one of those cases where some apt comments would explain
> the "why" and the method can just be the "what".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/C2/pull/715/files#r42449803>.
>